### PR TITLE
ContinuousFactor for accepting scipy.stats distributions

### PR DESCRIPTION
**Pull request by kris-singh**

### CODE REVIEW

**Comment by ankurankan** [comment]
@kris-singh Tests are failing. Please fix them.

**Comment by khalibartan** [comment]
@kris-singh Can you re-label the PR to something related to context of the problem which PR solves. It will help us in reviewing.

File: pgmpy/factors/continuous/ContinuousFactor.py [hunk 1]
```diff
@@ -4,17 +4,21 @@
 
 import numpy as np
 import scipy.integrate as integrate
+from scipy.stats import multivariate_normal, matrix_normal, dirichlet
+import scipy
```
Comment (ankurankan) [inline pgmpy/factors/continuous/ContinuousFactor.py:8]: @kris-singh Do we need both of these import statements?

**Comment by ankurankan** [comment]
@khalibartan @kris-singh @yashu-seth I am having second thoughts about this implementation. It will get too complicated and nested when we have a lot of distributions. I have created a new issue #782 for discussing a possible new architecture.

**Review by ankurankan — CHANGES_REQUESTED** [review]

**Comment by ankurankan** [comment]
@kris-singh We would like to have to code according to the new architecture that we discussed in #782. So closing this.

**Pull request closed without merging**